src/styles/blocks/header.scss

```diff
@@ -0,0 +1,17 @@
+.header {
+  background-color: #{$background-color};
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  display: flex;
+  justify-content: space-between;
+  padding: 0 50px;
+  height: #{$header-height};
+  align-items: center;
+  margin-bottom: 50px;
```

Comment:
According to BEM, blocks cannot have outer spacings. Your .header is a block. And it has margin-bottom. Let's fix this